app/assets/stylesheets/common.scss

```diff
@@ -4,6 +4,9 @@
 @import "rails_bootstrap_forms";
 @import "colors";
 
+$bootstrap-icons-font-dir: "bootstrap-icons/font/fonts";
+@import "bootstrap-icons/font/bootstrap-icons";
```

Comment:
Am I right in thinking that we've used icons from bootstrap icons before, but by manually importing them to our assets instead of like this?